### ISSUE TITLE
Vet Center audit

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17371]
## Description
Complete a full accessibility audit of the Vet Center product, including:
- [Main page](https://www.va.gov/boston-vet-center/)
- [Locations page](https://www.va.gov/boston-vet-center/locations/)

## Acceptance Criteria

- [ ] Accessibility audit has been completed
- [ ] Tickets have been created for recommended updates and improvements
- [ ] Any high priority/critical updates are brought to the attention of PM/DM/PO

**Comment by davidmpickett** [comment]
@jilladams FYI in scrum today we refined this and pulled it in to Sprint 1

**Comment by laflannery** [comment]
### Accessibility Updates
- Headings:
  - We have been working on reducing headings within the contact section of facility information. Phone/Hours/Address should follow this same pattern (whatever is ultimately decided)
  - We also should make sure that we don't unnecessarily use the VADS font size class on headings unless absolutely necessary. This is used on quite a few headings on the current implementation and we should make sure we remove this as it makes the visual hierarchy confusing.
  - I would recommend that “Need help after hours” is moved below “Hours”, as an H4 heading if heading hierarchy is kept
    - “Other locations” would move below this alert
- Alt text for “Static Map” images should be updated to be more meaningful
  - “Static map to get directions for…” seems duplicative of the link already on the page. I would like to consider how we can avoid this
- Update accordion intro text to remove the word “Click” and align with VBA (That template says "Select" instead".
  - There is a ticket for the [prepare for your visit section](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13937) but this also needs to be done on all the services sections as well

### Updates that should be made to align with upcoming DST changes:

**`<va-icon>` migration**
Planned for deprecation of current icons beginning of May
- Icon on Operating status alert
  - This is contained within the component so it might get auto updated


**Upcoming Heading/Link guidance**
No timeline yet for when [guidance](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2146) will be formalized
- Should the headings on Other nearby Vet Centers be unlinked and a separate CTA be added to align with upcoming guidance or is this an exception?

**Components**
- [Link](https://design.va.gov/components/link/)
  - Directions link because this doesn’t need to open in a new tab anymore
  - However, on the Locations page, the directions links need to be unique and the link component doesn’t yet have this flexibility. [It might in the future though](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1644).
- [Card](https://design.va.gov/components/card)
  - Update Spotlights to use the Card component, there is already [a ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17618) for this

### Other updates/improvements:
- “Non-traditional hours” text:
  - This entire text is bold and goes against VA content styles guidelines. This should be standard paragraph text
  - (This is sort of discussed above in the Heading section) This text and the following alert seems out of place - we have the hours, then we have the info about other locations then we have more content about hours? So maybe it’s more that the Other locations content is actually out of place.
- Find other locations link does **not** go directly to Vet Center param in the Facility Locator (I think some folks think that it does). We should potentially have it go [here](https://www.va.gov/find-locations/?facilityType=vet_center) though

**Comment by laflannery** [comment]
Audit is completed and these updates/recommendations should be considered when going through next steps for Vet Center 2.0

**Comment by laflannery** [comment]
@thejordanwood Here is [my comment](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17371#issuecomment-2043127994) with recommended update. Not sure how/when these might apply to your work but let me know if you have any questions
